src/Codec/Xlsx/Types/Common.hs

```diff
@@ -42,8 +41,8 @@ module Codec.Xlsx.Types.Common
   , DateBase(..)
   , dateFromNumber
   , dateToNumber
-  , int2col
-  , col2int
+  , columnIndexToText -- previously int2col
```

Comment:
what about marking previous ones as deprecated and making them aliases to the new ones before dropping in a later release?

Reply:
@LukeTemp do you still plan to work on this?

Reply:
Yes @qrilka I plan to get this finished tomorrow, I was having issues with resolving 3 test cases that I did not expect to be affected by these changes - I'll elaborate in my commit